site/docs/reference/Connectors/capture-connectors/MySQL/MySQL.md

```diff
@@ -29,6 +29,8 @@ To use this connector, you'll need a MySQL database setup with the following.
   to which the connector occasionally writes a small amount of data (a UUID,
   specifically) to ensure accuracy when backfilling preexisting table contents.
   - The default name is `"flow.watermarks"`, but this can be overridden in `config.json`.
+  - The watermark table will only ever have one row per capture from that database and that row is updated once per table during the initial backfill for MySQL databases.
```

Comment:
"Once per table" is a little bit ambiguous. In the steady state we update it once per backfill query, so it's more like "once per 50k rows backfilled" from a user perspective.

Reply:
Updated this